python/ray/serve/master.py

```diff
@@ -424,6 +426,25 @@ def _scale_replicas(self, backend_tag, num_replicas):
         current_num_replicas = len(self.replicas[backend_tag])
         delta_num_replicas = num_replicas - current_num_replicas
 
+        _, _, replicas_config = self.backends[backend_tag]
+        resource_per_replica = replicas_config.resource_dict
```

Comment:
I think this is only used once, would suggest just calling `scheduler.try_schedule(replica_config.resource_dict)`